### PR TITLE
chore: Fix rendering for admonitions with custom title

### PR DESCRIPTION
**Pull request by joonas**
## Feature or Problem

While browsing the docs, I noticed we had a couple of admonitions that were not rendering correctly due to having a custom title (presumably difference between `docusaurus 2.x` vs. `3.x`). This fixes those.

### CODE REVIEW

**Review by vados-cosmonic — APPROVED** [review]
LGTM :rocket:

**Comment by vados-cosmonic** [comment]
Hey @joonas looks like some changes made it in before this could be reviewed (and merged) -- would you mind fixing the conflicts? I'll merge right after that

**Comment by joonas** [comment]
@vados-cosmonic should be good now 😄

**Pull request merged**
